crates/core/tedge_api/src/workflow/log/logged_command.rs

```diff
@@ -162,15 +166,26 @@ impl LoggedCommand {
         };
 
         command
-            // TODO: should use tmp from config
-            .current_dir("/tmp")
+            .current_dir(tmp_dir)
             .stdin(Stdio::piped())
             .stdout(Stdio::piped())
             .stderr(Stdio::piped());
 
         Ok(LoggedCommand { command })
     }
 
+    pub fn from_command(mut command: std::process::Command, tmp_dir: impl AsRef<Path>) -> Self {
```

Comment:
`working_dir` would be more appropriate.

```suggestion
    pub fn from_command(mut command: std::process::Command, working_dir: impl AsRef<Path>) -> Self {
```

Reply:
Renamed in 10235e8e5580ee56c6284af7a49ec7d74f73ff28